content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx

```diff
@@ -59,41 +79,137 @@ const result = await prisma.posts.findMany({
+| `dog <-> fox`                           | No     | 'fox' doesn't follow 'dog' in the text  |
 
-The `search` field uses the database's native querying capabilities under the hood. This means that the Prisma Client [supports what the database supports](https://www.postgresql.org/docs/12/functions-textsearch.html).
+For the full range of supported operations, see the [PostgreSQL full text search documentation](https://www.postgresql.org/docs/12/functions-textsearch.html).
+
+### MySQL
+
+The following examples demonstrate use of the MySQL 'and' (`+`) and 'not' (`-`) operators:
+
+```ts
+// All posts that contain the words 'cat' or 'dog'.
+const result = await prisma.posts.findMany({
+  where: {
+    body: {
+      search: 'cat dog',
+    },
+  },
+})
+
+// All posts that contain the words 'cat' and not 'dog'.
+const result = await prisma.posts.findMany({
+  where: {
+    body: {
+      search: '+cat -dog',
+    },
+  },
+})
+
+// All drafts that contain the words 'cat' and 'dog'.
+const result = await prisma.posts.findMany({
+  where: {
+    status: 'Draft',
+    body: {
+      search: '+cat +dog',
+    },
+  },
+})
+```
 
 To get a sense of how the query format works, consider the following text:
 
 **"The quick brown fox jumps over the lazy dog"**
 
 Here's how the following queries would match that text:
 
-| Query                                   | Match? | Description                         |
-| :-------------------------------------- | :----- | :---------------------------------- |
-| `fox & dog`                             | Yes    | The text contains fox and dog       |
-| `dog & fox`                             | Yes    | The text contains dog and fox       |
-| `dog & cat`                             | No     | The text doesn't contain cat        |
-| `!cat`                                  | Yes    | There is not cat in the text        |
-| <inlinecode>fox &#124; cat</inlinecode> | Yes    | The text contains fox or cat        |
-| <inlinecode>cat &#124; pig</inlinecode> | No     | The text doesn't contain cat or pig |
-| `fox <-> dog`                           | Yes    | dog follows fox in the text         |
-| `dog <-> fox`                           | No     | dog doesn't follow fox in the text  |
+| Query       | Match? | Description                             |
+| :---------- | :----- | :-------------------------------------- |
+| `+fox +dog` | Yes    | The text contains 'fox' and 'dog'       |
+| `+dog +fox` | Yes    | The text contains 'dog' and 'fox'       |
+| `+dog +cat` | No     | The text contains 'dog' but not 'cat'   |
+| `-cat`      | Yes    | 'cat' is not in the text                |
+| `fox dog`   | Yes    | The text contains 'fox' or 'cat'        |
+| `-cat -pig` | No     | The text doesn't contain 'cat' or 'pig' |
+
+For the full range of supported operations, see the [MySQL full text search documentation](https://dev.mysql.com/doc/refman/8.0/en/fulltext-boolean.html).
```

Comment:
Is this the right MySQL version and page?

Reply:
I cannot find a newer version so I think so.